whats_fresh/whats_fresh_api/templates/vendor.html

```diff
@@ -18,6 +18,16 @@ <h4 class='light accent'>{{ message }}</h4>
 {% load static %}
 
 {% block content %}
+
+<head>
```

Comment:
Get rid of this entire <head> section. It's only being used for the value of dbAddress. Instead, why not render dbAddress directly? (as shown in another comment)